mteb/abstasks/AbsTask.py

```diff
@@ -216,7 +216,9 @@ def calculate_metadata_metrics(
                     pbar_lang.set_postfix_str(f"Language: {lang}")
                     print(f"Processing metadata for language {lang}")
                     split_details = self._calculate_metrics_from_split(split, lang)
-                    all_details[split][lang] = split_details
+                    all_details[split]["hf_subset_descriptive_stats"][lang] = (
```

Comment:
```suggestion
                    all_details[split]["hf_subset_descriptive_stats"][hf_subset] = (
```
As it is not nec. language